pages/common/pyenv.md

```diff
@@ -0,0 +1,27 @@
+# pyenv  
+
+> Switch between multiple versions of Python easily.
+
+- List all available commands:
+
+`pyenv commands`
+
+- List all Python versions known to pyenv:
+
+`pyenv versions`
+
+- Install a Python version known to pyenv, default in ~/.pyenv/versions directory:
+
+`pyenv install {{version}}`
+
+- Uninstall a Python version known to pyenv:
+
+`pyenv uninstall {{version}}`
+
+- Set Python version to be used globally in the current machine:
+
+`pyenv global {{version}}`
+
+- Set Python version to be used in the current directory recursively:
```

Comment:
suggestion: "recursively" --> "and all directories below it". Just to make it a tad more explicit :)